src/models/darp10/repairs.md

```diff
@@ -0,0 +1,287 @@
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+8. Carefully replace the heatsink.
+9. Replace the seven heatsink/fan screws in order, starting with #1, then #2, and so on until #7.
+
+## Replacing the speakers:
+
+The system has two bottom-firing speakers, which can be replaced individually.
+
+**Part numbers:**
+- 16" model (darp10):
+    - Left speaker:
+        - Model number: `V560TU-L`
+        - Part number: `6-23-5V56T-0L1`
+    - Right speaker:
+        - Model number: `V560TU-R`
+        - Part number: `6-23-5V56T-0R1`
+- 14" model (darp10-b):
+    - Left speaker:
+        - Model number: `V540TU-L`
+        - Part number: `6-23-5V54T-0L1`
+    - Right speaker:
+        - Model number: `V540TU-R`
+        - Part number: `6-23-5V54T-0R0`
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 10 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the speakers:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove any clear tape securing the speaker wires.
+3. Disconnect the black speaker connectors from the motherboard.
+    - This can alternatively be done after freeing the speakers.
+4. Remove the speakers from the chassis.
+    - For the 16" model (darp10), unscrew the speakers. To help free the left speaker, [remove the main battery](#replacing-the-battery).
+    ![Speaker wires and screws (darp10, 16")](./img/speakers-16.webp)
+    - For the 14" model (darp10-b), lift the speakers up and off of the plastic posts.
+    ![Speaker wires and posts (darp10-b, 14")](./img/speakers-14.webp)
+5. Slide the new speakers onto the plastic posts and connect them to the motherboard.
+    - The left speaker's wire can be run through the channel in the chassis before or after plugging it in.
+
+## Replacing the keyboard:
+
+The keyboard can be replaced if its switches or electronics have been damaged.
+
+**Part numbers:**
+- 16" (darp10) keyboard: `6-23-RV56T-011`
+- 14" (darp10-b) keyboard: `6-23-RL245-010`
+- Keyboards shipped by System76 include a custom-printed Super key.
+
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the keyboard:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the keyboard screws (highlighted yellow below).
+    - On the 16" model (darp10), there is one keyboard screw.
+    ![Keyboard screws (16")](./img/keyboard-screw-16.webp)
+    - On the 14" model (darp10-b), there are two keyboard screws.
+    ![Keyboard screws (14")](./img/keyboard-screws-14.webp)
+3. For the 16" model (darp10) only, you can optionally [remove the bottom panel](), [remove the heatsink](), and slowly pull the tabs highlighted below to remove the two keyboard adhesive strips.
```

Comment:
These bottom panel and heatsink links aren't populated

Reply:
Fixed.